PinpointKit/PinpointKit/Sources/Core/FeedbackTableViewDataSource.swift

```diff
@@ -47,15 +47,21 @@ final class FeedbackTableViewDataSource: NSObject, UITableViewDataSource {
     // MARK: - FeedbackTableViewDataSource
     
     private static func sectionsFromConfiguration(_ interfaceCustomization: InterfaceCustomization, screenshot: UIImage, logSupporting: LogSupporting, userEnabledLogCollection: Bool) -> [Section] {
-        guard logSupporting.logCollector != nil else { return [] }
+        var sections = [Section]()
```

Comment:
For some reason I prefer `var sections: [Section] = []` but that might just be me.